airflow/www/views.py

```diff
@@ -4534,6 +4535,14 @@ def action_set_retry(self, tis):
         self.update_redirect()
         return redirect(self.get_redirect())
 
+    @action('set_skipped', "Set state to 'skipped'", '', single=False)
+    @action_has_dag_edit_access
+    def action_set_skipped(self, tis):
+        """Set state to skipped."""
+        self.set_task_instance_state(tis, TaskInstanceState.SKIPPED)
```

Comment:
```suggestion
        self.set_task_instance_state(tis, State.SKIPPED)
```
`TaskInstanceState` was undefined when I was testing locally